open-metadata-resources/open-metadata-samples/open-metadata-security-samples/pom.xml

```diff
@@ -27,6 +27,21 @@
         Samples showing how to configure an OMAG Service Platform for security.
     </description>
 
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-site-plugin</artifactId>
+                <version>3.9.1</version>
```

Comment:
Should remove the explicit version
All version numbers should be in the dependencyManagement section in the top level pom, and then the version line dropped here - same with 3.1.1 below

In fact we already have those two artifacts in that list, so the fix here is to simply remove the version lines.
We will inherit the version constraint from the parent project & keep them consistent within the egeria tree

Reply:
I tried tackling this suggestion first as it was just a matter of removing a line, I ran into some issue with it. I will look further into this!

Maven was not happy with the removal of the version, definitely need to figure out why.